plugins/org.eclipse.xtext.ui.shared/plugin.properties

```diff
@@ -2,7 +2,7 @@ OpenModelElementCommand.name=Open Model Element
 OpenModelElementCommand.description=Open the declaration of an Xtext element
 XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
-XtextNatureAddAction.label=Add Xtext Nature
-XtextNatureRemoveAction.label=Remove Xtext Nature
+XtextNatureAddAction.label=Convert to Xtext project
+XtextNatureRemoveAction.label=Unconfigure Xtext
```

Comment:
Should we remove this all together? Other plugins doesn't seem to support removing as well.